src/App.tsx

```diff
@@ -122,7 +130,11 @@ export const App: React.FC = () => {
 
         {/* Hide the footer if there are no todos */}
         {todos && todos.length > 0 && (
-          <Footer setQuery={setQuery} query={query} left={returnLeftNumber} />
+          <Footer
+            handleSetQuery={handleSetQuery}
```

Comment:
Follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions